vars/RedHat.yml

```diff
@@ -0,0 +1,4 @@
+---
+timesync_required_packages:
+  - rpm-python
+  - libselinux-python
```

Comment:
I don't think any of these changes are required - please revert this file

Reply:
In addition - this is the source of the rhel8 test failures, because these packages do not exist on rhel8